CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Reimplemented `probe()` / `probe_all()` methods of the testkit with
   a revertible database. (#8)
+- Renamed `public_mount` and `private_mount` to `public_handler` and `private_handler`. (#23)  
```

Comment:
I know this is beyond nitpicking, but: I'd formulate

> Renamed `public_mount` and `private_mount` in `TestKitApi` to `public_handler` and `private_handler`, respectively, and made these methods return `iron::Chain` references.

in order to fully reflect changes brought by this PR.